src/Nethermind/Nethermind.Evm/CallResult.cs

```diff
@@ -67,7 +67,7 @@ private CallResult(EvmExceptionType exceptionType)
         public bool ShouldRevert { get; }
         public bool? PrecompileSuccess { get; }
         public bool IsReturn => StateToExecute is null;
-        public bool IsException => ExceptionType != EvmExceptionType.None;
+        public bool IsException => ExceptionType != EvmExceptionType.None && ExceptionType != EvmExceptionType.Revert;
```

Comment:
We should add some comment here to explain the reasoning as to why `Revert` is not treated as an exception for future maintainers.